src/api/mediators.coffee

```diff
@@ -9,6 +9,34 @@ atna = require 'atna-audit'
 utils = require "../utils"
 auditing = require '../auditing'
 
+maskPasswords = (defs, config) ->
+  if not config
+    return
+
+  defs.forEach (d) ->
+    if d.type is 'password' and config[d.param]
+      if d.array
+        config[d.param] = config[d.param].map -> '**********'
```

Comment:
I think it would be better to pull the *'s string out into a variable as it gets duplicated in a few places (something like `mask`?)